ml-agents-envs/mlagents_envs/side_channel/default_training_analytics_side_channel.py

```diff
@@ -0,0 +1,49 @@
+import sys
+import uuid
+import mlagents_envs
+
+from mlagents_envs.exception import UnityCommunicationException
+from mlagents_envs.side_channel import SideChannel, IncomingMessage, OutgoingMessage
+from mlagents_envs.communicator_objects.training_analytics_pb2 import (
+    TrainingEnvironmentInitialized,
+)
+from google.protobuf.any_pb2 import Any
+
+
+class DefaultTrainingAnalyticsSideChannel(SideChannel):
+    """
+    Side channel that sends information about the training to the Unity environment so it can be logged.
+    """
+
+    def __init__(self) -> None:
+        # >>> uuid.uuid5(uuid.NAMESPACE_URL, "com.unity.ml-agents/TrainingAnalyticsSideChannel")
+        # UUID('b664a4a9-d86f-5a5f-95cb-e8353a7e8356')
+        # We purposefully use the SAME side channel as the TrainingAnalyticsSideChannel
+        super().__init__(uuid.UUID("b664a4a9-d86f-5a5f-95cb-e8353a7e8356"))
```

Comment:
The uuid between DefaultTrainingAnalyticsSideChannel and TrainingAnalyticsSideChannel must be the same so they are interpreted the same way in C#. A more elegant solution would be to have TrainingAnalyticsSideChannel inherit from DefaultTrainingAnalyticsSideChannel

Reply:
change to super().__init__ with parent class.